app/components/contact-form.tsx

```diff
@@ -1,17 +1,94 @@
-import { Todo } from "./todo";
+import type { HTMLInputTypeAttribute } from "react";
+import { useFetcher } from "@remix-run/react";
+import type { SerializeFrom } from "@remix-run/server-runtime";
 
-interface Props {
+import { Button } from "~/components/button";
+import { Todo } from "~/components/todo";
+import type { action as contactAction } from "~/routes/contact";
+
+export const ContactForm: React.FC<{
   title: React.ReactNode;
-}
-export const ContactForm = ({ title }: Props) => {
+}> = ({ title }) => {
+  const fetcher = useFetcher<SerializeFrom<typeof contactAction>>();
+  const isSuccess = fetcher.type === "done" && !fetcher.data;
+  return (
+    <article className="bg-secondary flex flex-col items-center text-white w-full -mb-[50px] py-[3vh] px-[100px] mx-auto">
+      <section className="text-center">
+        <div className="text-3xl font-semibold text-peach">
+          {isSuccess ? "Takk for din interesse!" : title}
+        </div>
+        <span>
+          {isSuccess
+            ? "Vi tar kontakt med deg så snart som mulig."
+            : "Fyll ut skjemaet så kontakter vi deg!"}
+        </span>
+      </section>
+      <section className="flex justify-between w-full max-w-7xl gap-12 mb-10">
+        <fetcher.Form
+          method="post"
+          action="/contact"
+          className="w-[50%]"
+          name="contact"
+          data-netlify="true"
+          data-netlify-honeypot="bot-field"
```

Comment:
:question: _I have a question / can you clarify?_ [ref](https://devblogs.microsoft.com/appcenter/how-the-visual-studio-mobile-center-team-does-code-review/)
Disse blir vell ikke brukt til noen ting når vi ikke generer statisk html? 🤔 

https://docs.netlify.com/forms/setup/
> Netlify comes with built-in form handling that’s enabled by default. Our build bots do it by parsing your **HTML files directly at deploy time**